architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -175,29 +175,151 @@ endif::[]
+      14 seconds ago
+
+35
+  tagged from centos/python-35-centos7
+
+  * centos/python-35-centos7@sha256:2efb79ca3ac9c9145a63675fb0c09220ab3b8d4005d35e0644417ee552548b10
+      7 seconds ago
+----
+
+Using image streams has several significant benefits:
+
+* You can tag, rollback a tag, and quickly deal with images, without having to re-push using the command line.
+
+* You can trigger Builds and Deployments when a new image is pushed to the registry. Also,
+{product-title} also has generic triggers for other resources (such as Kubernetes objects).
+
+* You can xref:../../dev_guide/managing_images.adoc#importing-tag-and-image-metadata[mark a tag for periodic re-import].
+If the source image has changed, that change is picked up and reflected in the image stream and can trigger the Build and/or Deployment flow, depending upon the Build or Deployment configuration.
+
+* You can share images using fine-grained access control and quickly distribute images across your teams.
+
+* If the source image changes, the image stream tag will still point to a known-good version of the image, ensuring that your application will not break unexpectedly.
+
+* You can configure security around who can view and use the images through permissions on the image stream objects. 
+
+* Users that lack permission to read or list images on the cluster level can still retrieve the images tagged in a project using image streams.
+
+
+For a curated set of image streams, see the
 link:https://github.com/openshift/library[OpenShift Image Streams and Templates
 library].
 
-For example, if a deployment is using a certain image and a new version of that
-image is created, a deployment could be automatically performed.
+When using image streams, it is important to understand what the image stream tag is pointing to and how changes to tags and images can affect you. For example:
+
+* If your image stream tag points to a Docker image tag, you need to understand how that Docker image tag is updated. For example, a Docker image tag `docker.io/ruby:2.4` will probably always point to a v2.4 ruby image. But, a Docker image tag `docker.io/ruby:latest` will probably change with major versions. So, the Docker image tag that a image stream tag points to can tell you how stable the image stream tag will be, if you choose to reference it.
+
+* If your image stream tag follows another image stream tag (it does not point directly to a docker image tag), it is possible that the image stream tag will be updated to follow a different image stream tag in the future. Again, this could result in picking up an incompatible version change.
+
+=== Important terms
+
+Docker repository::
+A central location for uploading and downloading images.
+For example, the OpenShift Jenkins images are in a Docker registry:
++
+----
+docker.io/openshift/jenkins-2-centos7
+----
+
+Docker image::
+A reference to an image for a given external registry. It uses the standard Docker pull specification for its name.
```

Comment:
Docker image:  A specific set of content that can be run as a container.  Usually associated with a particular tag within a docker repository.